test/e2e/features/basic.feature

```diff
@@ -135,6 +135,10 @@ Feature: Basic test
         And unsetting config property "enable-cluster-monitoring" succeeds
         And unsetting config property "memory" succeeds
 
+    @linux
+    Scenario: Bundle generation check
+        Given executing "crc bundle generate" succeeds
```

Comment:
I forgot if it's possible to add comments in .feature files? I'd mention that after running `crc bundle generate`, the pull secret has been removed from the VM, as this could have impacts on other tests if this ever gets moved around?

Reply:
@cfergeau Sure I will add that comment here.

Reply:
@cfergeau Added comment.